app/controllers/user.js

```diff
@@ -69,18 +69,20 @@ exports.signUp = (req, res, next) => {
 };
 
 exports.listUsers = (req, res, next) => {
-  const auth = req.headers.authorization; // auth is in base64(username:password)  so we need to decode the base64
-  try {
-    const decoded = sessionsManager.decode(auth);
-  } catch (e) {
-    throw errors.unauthorizedNoLogin;
-  }
-  User.getUsers(req.query.page, req.query.limit)
-    .then(users => {
-      res.status(200).send(users);
-    })
-    .catch(err => {
-      logger.error('Error looking for users in the database');
-      next(err);
-    });
+  User.count().then(total => {
```

Comment:
the idea of this count is great, but sequelize gives us some help and has `findAndCountAll` which fits perfectly in these cases.